values/src/main/proto/spine/time/time.proto

```diff
@@ -82,4 +84,16 @@ message OffsetDateTime {
     LocalDate date = 1;
     LocalTime time = 2;
     ZoneOffset offset = 3;
-}
\ No newline at end of file
+}
+
+// An interval between two points in time.
+message Interval {
+    // A start of the interval.
+    google.protobuf.Timestamp start = 1;
+
+    // An end of the interval.
+    google.protobuf.Timestamp end = 2;
+
+    // A duration between `start` and `end`.
```

Comment:
The duration can be calculated. We have all the needed information for this. There's no reason to store the duration. `Intervals` should have a conversion method, `toDuration(Interval)`.